### PR TITLE
Peter Duang修改过

### PR DESCRIPTION
**Pull request by prometheosduan**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2490)

### CODE REVIEW

**Comment by prometheosduan** [comment]
@casey-collab Please Review It:)

File: Chinese/Peter/Peter.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+孙悟空变成了recipe，被fork了好几千次。
```
Comment (casey-collab) [inline Chinese/Peter/Peter.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
It looks like the file `Chinese/Peter/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by prometheosduan** [comment]
@casey-collab      Please review it.

File: Chinese/Peter/Peter.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/Peter/Peter.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline Chinese/Peter/Peter.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
It looks like the file `Chinese/Peter/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by prometheosduan** [comment]
@casey-collab Please review it.

File: Chinese/Peter/Peter.md (hunk 1 above)
Comment (casey-collab) [inline Chinese/Peter/Peter.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
It looks like the file `Chinese/Peter/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by prometheosduan** [comment]
@casey-collab Please review it.

**Comment by casey-collab** [comment]
Looks great! :cherry_blossom:

**Pull request merged**